### PR TITLE
Changes the doc template

### LINKED ISSUE #209: Improve CSS documentation
- Rethink the documentation structure
- Improve the usage table
- Create an accessibility table
- Style the documentation section
- Write guides of how to document.
- Update the template(s)
- Go back and change what’s done to follow this new standards

### PR DESCRIPTION
**Pull request by andresgalante**
This PR closes #209 and improves the CSS documentation template.

### CODE REVIEW

File: build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md [hunk 1]
```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-Write accessibility, aria tags, tab index and other notes to make this component accessible.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `{{bemName}}` |  accecibility notes. |
```
Comment (srambach) [inline build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md]: "accessibility"

File: build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md [hunk 2]
```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
...
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.class-name-here` **Applied to:** `<tags-here>` |  **Outcome:** Outcome-here **Required:** Yes/No **Remarks:** Remarks-here |
-| Example: `.btn` **Applied to:** `<a>`, `<button>` |  **Outcome:** Initiates a button **Required:** Yes **Remarks:** Always use it with a modifier class. |
-| `.btn-secondary` **Applied to:** `.btn` | **Outcome:** Adds secondary color **Required:** Yes |
\ No newline at end of file
+| Class | Applied | Outcomete |
```
Comment (srambach) [inline build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md]: Should it say "Applied to"?
Also, "Outcome"

File: build/blueprints/layout/files/src/patternfly/layouts/__moduleName__/docs/code.md [hunk 3]
```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
...
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.class-name-here` | **Applied to:** `<tags-here>` |  **Outcome:** Outcome-here | **Required:** Yes/No | **Remarks:** Remarks-here |
-| Example: `.btn` | **Applied to:** `<a>`, `<button>` |  **Outcome:** Initiates a button | **Required:** Yes | **Remarks:** Always use it with a modifier class. |
-| `.btn-secondary` | **Applied to:** `.btn` | **Outcome:** Adds secondary color | **Required:** Yes |
\ No newline at end of file
+| Class | Applied | Outcomete |
```
Comment (srambach) [inline build/blueprints/layout/files/src/patternfly/layouts/__moduleName__/docs/code.md]: "Outcome" here too

File: build/blueprints/pattern/files/src/patternfly/patterns/__moduleName__/docs/code.md [hunk 4]
```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
...
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.class-name-here` **Applied to:** `<tags-here>` |  **Outcome:** Outcome-here **Required:** Yes/No **Remarks:** Remarks-here |
-| Example: `.btn` **Applied to:** `<a>`, `<button>` |  **Outcome:** Initiates a button **Required:** Yes **Remarks:** Always use it with a modifier class. |
-| `.btn-secondary` **Applied to:** `.btn` | **Outcome:** Adds secondary color **Required:** Yes |
\ No newline at end of file
+| Class | Applied | Outcomete |
```
Comment (srambach) [inline build/blueprints/pattern/files/src/patternfly/patterns/__moduleName__/docs/code.md]: "Outcome" again

File: build/blueprints/layout/files/src/patternfly/layouts/__moduleName__/docs/code.md [hunk 5]
```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-Write accessibility, aria tags, tab index and other notes to make this component accessible.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `{{bemName}}` |  accecibility notes. |
```
Comment (srambach) [inline build/blueprints/layout/files/src/patternfly/layouts/__moduleName__/docs/code.md]: "accessibility"

File: build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md (hunk 1 above)
Comment (srambach) [inline build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md]: "accessibility"

File: build/blueprints/pattern/files/src/patternfly/patterns/__moduleName__/docs/code.md [hunk 6]
```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-Write accessibility, aria tags, tab index and other notes to make this component accessible.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `{{bemName}}` |  accecibility notes. |
```
Comment (srambach) [inline build/blueprints/pattern/files/src/patternfly/patterns/__moduleName__/docs/code.md]: "accessibility"

File: src/patternfly/components/Alert/docs/code.md [hunk 7]
```diff
@@ -3,14 +3,17 @@
 Always add a modifier class. Do not use `.pf-c-alert` on it's own.
```
Comment (srambach) [inline src/patternfly/components/Alert/docs/code.md]: Should be "its" not "it's"

File: src/patternfly/components/Content/docs/code.md [hunk 8]
```diff
@@ -14,6 +14,6 @@ This component is an exception to the variable system since we style type select
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.pf-c-content` **Applied to:** `<div>`, `<section>` or `<article>` |  **Outcome:** Generates vertical rythm and typographic treatment to html elements **Required:** Yes |
\ No newline at end of file
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-c-content` | `<div>`, `<section>` or `<article>` | Generates vertical rythm and typographic treatment to html elements |
```
Comment (srambach) [inline src/patternfly/components/Content/docs/code.md]: "rhythm"

File: src/patternfly/layouts/Split/docs/code.md [hunk 9]
```diff
@@ -1,15 +1,12 @@
...
-| `.pf-l-split__primary` | **Outcome:** Specificies the primary child of the layout, which expands vertucally when needed. The primary child will expand horizontally to fill all space not needed by the secondary children. | **Required:** No | **Remarks:** If the primary is not explicitly set, the second child with act as the primary child.|
-| `.pf-l-split__secondary` | **Outcome:** Specifies a secondary child of the layout, which accommodates to its content. | **Required:** No **Remarks:** A split can have one or more secondary children. If secondary children are not explicitly set, all children but the second will act as secondary children.|
-| `.pf-has-gutter` | **Outcome:** Adds space between children | **Required:** No |
-
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-l-split` | `<div>`, `<section>` or `<article>` | Initiates the split layout |
+| `.pf-l-split__primary` | `<div>` | Specificies the primary child of the layout, which expands vertucally when needed. The primary child will expand horizontally to fill all space not needed by the secondary children. If the primary is not explicitly set, the second child with act as the primary child.|
```
Comment (srambach) [inline src/patternfly/layouts/Split/docs/code.md]: "vertically"
at the end, "...the second child will act as the primary child. |"

File: src/patternfly/layouts/Level/docs/code.md [hunk 10]
```diff
@@ -4,8 +4,8 @@ A level is a layout that distributes the space between children evenly and it ce
...
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.pf-l-level` |  **Outcome:** Initializes the level layout **Required:** Yes |
-| `.pf-l-level__item` |  **Outcome:** Explicitly sets a child of the level | **Required:** No | **Remarks:** This class isn't necessary, but it is included to keep inline with BEM convention, and to provide an entity that will later be used for applying modifiers.
-| `.pf-has-gutter` |  **Outcome:** Adds space between children | **Required:** No
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-l-level` | `<div>`, `<section>` or `<article>` | Initializes the level layout |
```
Comment (srambach) [inline src/patternfly/layouts/Level/docs/code.md]: Oxford comma after `section`

File: src/patternfly/layouts/Split/docs/code.md [hunk 11]
```diff
@@ -1,15 +1,12 @@
...
-| Class | Usage |
-| -- | -- |
-| `.pf-l-split` | **Outcome:** Initiates the split layout | **Required:** Yes |
-| `.pf-l-split__primary` | **Outcome:** Specificies the primary child of the layout, which expands vertucally when needed. The primary child will expand horizontally to fill all space not needed by the secondary children. | **Required:** No | **Remarks:** If the primary is not explicitly set, the second child with act as the primary child.|
-| `.pf-l-split__secondary` | **Outcome:** Specifies a secondary child of the layout, which accommodates to its content. | **Required:** No **Remarks:** A split can have one or more secondary children. If secondary children are not explicitly set, all children but the second will act as secondary children.|
-| `.pf-has-gutter` | **Outcome:** Adds space between children | **Required:** No |
-
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-l-split` | `<div>`, `<section>` or `<article>` | Initiates the split layout |
```
Comment (srambach) [inline src/patternfly/layouts/Split/docs/code.md]: Comma after `<section>`

File: src/patternfly/layouts/Stack/docs/code.md [hunk 12]
```diff
@@ -6,9 +6,10 @@ child will expand vertically to fill the container if needed.
...
 
-| Class | Usage |
-| -- | -- |
-| `.pf-l-stack` | **Outcome:** Initiates the stack layout | **Required:** Yes |
-| `.pf-l-stack__primary` | **Outcome:** Specifies the primary child of the layout, which expands vertically when needed. | **Required:** No |
-| `.pf-l-stack__secondary` | **Outcome:** Specifies a secondary child of the layout, which accommodates its content. | **Required:** No | **Remarks:** A stack can have one or more secondary children |
-| `.pf-has-gutter` | **Outcome:** Adds space between children | **Required:** No |
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-l-stack` | `<div>`, `<section>` or `<article>` | Initiates the stack layout |
```
Comment (srambach) [inline src/patternfly/layouts/Stack/docs/code.md]: Comma after `<section>`

**Comment by srambach** [comment]
Looks great! I just left a bunch of fiddly spelling/grammar comments. I would like "Applied to" better than "Applied" throughout, and "Outcome**te**" needs to be fixed throughout all.
This is great and makes it much more readable! Thank you!

File: build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md (hunk 2 above)
Comment (jgiardino) [reply build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md]: I found myself wanting to use "Applied To" as well. I think that would go for both tables.

File: src/patternfly/components/Button/docs/code.md [hunk 13]
```diff
@@ -4,21 +4,22 @@ Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-An active state button will appear pressed when active. There’s no need to add a class to `<button>`s as they use a pseudo-class. However, if you are using a button as a toggle, you can still force the same active appearance with `pf-is-active` (and include the `aria-pressed="true"`) should you need to replicate the state programmatically.
-
-Disabled buttons should include the `disabled="disabled"` attribute to indicate the state of the element to assistive technologies.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `disabled="disabled"` | `.pf-c-button` | Indicates the disabled state of the element to assistive technologies |
```
Comment (jgiardino) [inline src/patternfly/components/Button/docs/code.md]: Something to note about this attribute, is that we we expand this component so that these styles can be applied to `<a>`, then we would want to specify that this attribute applies to the `<button>` element only. In the menu, I include both buttons and links as menu item examples, and handled the different attributes as shown here: https://deploy-preview-206--pf-next.netlify.com/components/Dropdown/examples/#accessibility

I realize this comment is not part of the scope of this PR, but I wonder if we should at least mention the `<button>` element somewhere in this row?

File: src/patternfly/components/Button/docs/code.md [hunk 14]
```diff
@@ -4,21 +4,22 @@ Always add a modifier class to add color to the button.
...
+| `aria-pressed="true or false"` | `.pf-c-button` | If you are using a button as a toggle, you can still force the same active appearance with `pf-is-active` (and include the `aria-pressed=""`) should you need to replicate the state programmatically. |
```
Comment (jgiardino) [inline src/patternfly/components/Button/docs/code.md]: In the context of this table, I'd reword **Outcome** to something like this:

> Indicates that the button is a toggle. When set to "true", `pf-is-active` should also be set so that the button displays in an active state.

File: src/patternfly/layouts/Bullseye/docs/code.md [hunk 15]
```diff
@@ -1,10 +1,10 @@
...
-A bullseye layout is when there is a single child element, and that child is centered both vertically and horizontally in the parent.
+Use a bullseye layout when there is a single child element, and that child is centered both vertically and horizontally in the parent.
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.pf-l-bullseye` |  **Outcome:** Initializes the bullseye layout | **Required:** Yes | **Remarks:** A bullseye can only have one child
-| `.pf-l-bullseye__item` |  **Outcome:** Explicitly sets the child for the bullseye  | **Required:** No | **Remarks:** This class isn't necessary, but it is included to keep inline with BEM convention, and to provide an entity that will later be used for applying modifiers.
+| Class | Applied | Outcomete |
```
Comment (jgiardino) [inline src/patternfly/layouts/Bullseye/docs/code.md]: "Outcome"

Comment (jgiardino) [reply src/patternfly/layouts/Bullseye/docs/code.md]: Oh, there are a lot of "Outcomete"s. Just do a find all ;-)

File: src/patternfly/layouts/Bullseye/docs/code.md [hunk 16]
```diff
@@ -1,10 +1,10 @@
...
+| -- | -- | -- |
+| `.pf-l-bullseye` | `<div>` | Initializes the bullseye layout. A bullseye can only have one child. |
```
Comment (jgiardino) [inline src/patternfly/layouts/Bullseye/docs/code.md]: Throughout the docs/code.md files, there is a mix of "Initializes" and "Initiates". I'm not even sure if one is more appropriate than the other in this context, but should we be consistent? (I'm not in the dropdown docs, I used "defines")

File: src/patternfly/components/Button/docs/code.md [hunk 17]
```diff
@@ -4,21 +4,23 @@ Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-An active state button will appear pressed when active. There’s no need to add a class to `<button>`s as they use a pseudo-class. However, if you are using a button as a toggle, you can still force the same active appearance with `pf-is-active` (and include the `aria-pressed="true"`) should you need to replicate the state programmatically.
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `disabled="disabled"` | `.pf-c-button` | Indicates the disabled state of the `button` to assistive technologies |
+| `aria-pressed="true or false"` | Indicates that the button is a toggle. When set to "true", `pf-is-active` should also be set so that the button displays in an active state. |
```
Comment (srambach) [inline src/patternfly/components/Button/docs/code.md]: Needs one more table cell for Applied To
